README.md

```diff
@@ -45,7 +45,6 @@ your integration.
 - _[Enrich](https://platform.fullcontact.com/docs/apis/enrich/introduction)_
```

Comment:
Can you update these links, all are now under docs.fullcontact.com

Reply:
Updated for Enrich, Permission and Resolve